# Status of Emergency stop switch in PACMod system 

std_msgs/Header header

bool estop_status  # True if the switch is pressed (System is E-stopped)
bool estop_fault   # True if a fault has occurred in the switch